release/README.md

```diff
@@ -70,8 +70,15 @@ First, make sure you are connected to the production cluster (**note:** this req
 ```
 gcloud container clusters get-credentials bank-of-anthos-master --zone us-west1-a --project bank-of-anthos
 ```
+1. ***[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled production cluster***
 
-Then, you can simply apply the new manifest versions on top of the current environment:
+Currently the `bank-of-anthos-master` cluster has [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. Thus, when deploying to this cluster the pod service account (`boa-ksa-master`) used by _Workload Identity_ must be used as the serviceAccount in the manifests.
+
+Follow the steps `3` & `4` of the the [workload identity setup](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/docs/workload-identity.md) with `boa-ksa-master` as the `KSA_NAME` to deploy into production.
+
+2. ***Non Workload Identity Cluster***
```

Comment:
Minor, but since all the other H2+ headers are not all-capitalized, let's keep it consistent
```suggestion
2. ***Non Workload Identity cluster***
```

Reply:
Fixed in https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/462/commits/9c8629e8e08e3607be10db11b5ff108e879ba477